modules/021-cni-cilium/templates/configmap.yaml

```
@@ -167,7 +167,7 @@ data:
 
   write-cni-conf-when-ready: /host/etc/cni/net.d/05-cilium.conflist
 
-  {{- if .Values.cniCilium.internal.isIstioCNIEnabled }}
+  {{- if .Values.cniCilium.internal.exclusiveCNIPlugin }}
   cni-exclusive: "false"
   {{- else }}
   cni-exclusive: {{ .Values.cniCilium.exclusiveCNIPlugin | quote }}
```

```suggestion
  cni-exclusive: {{ .Values.cniCilium.internal.exclusiveCNIPlugin | quote }}
```
Let's do smth like this.